### ISSUE TITLE
Add a 'quiet' option

### ISSUE THREAD

**Issue by jonadem** [epsilonrt/mbpoll#11]
In some use cases, it could be nice if the output will be as concise as possible.

A '-q' option is not yet used.

**Comment by epsilonrt** [comment]
Yes, I do not see much interest but I would take care of it occasionally.

**Comment by jonadem** [comment]
The use case is have in mind is when you want to call the application from a script and monitor several slaves and addresses in a specific way.

**Comment by eliafino** [comment]
Please, quite option
